modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -196,70 +196,104 @@ template<typename Net> class Params {
     detail::ParamDesc desc;
 };
 
+/*
+* @brief This structure provides functions for generic network type that
+* fill inference parameters.
+* @see struct Generic
+*/
 template<>
 class Params<cv::gapi::Generic> {
 public:
+    /** @brief Class constructor.
+
+    Constructs Params based on model information and sets default values for other
+    inference description parameters. Model is loaded and compiled with OpenVINO.
+
+    @param tag string tag of the network for which these parameters are intended.
+    @param model path to topology IR (.xml file).
+    @param weights path to weights (.bin file).
+    @param device device specifier.
```

Comment:
I got comment, that this should be the `target device to use`.

Reply:
Done